addonscript/requirements/base.txt

```diff
@@ -51,9 +51,9 @@ aiomemoizettl==0.0.3 \
     --hash=sha256:07a6becac60f6cd2604b9f2b73bcd9a50079a0b7b55e2a4e45b1eec5a3ea9659 \
```

Comment:
I'd love for all the .txt files to have the py3x as part of the name for consistency, but that doesn't have to happen in this PR.

An observation that doesn't require action: our requirements.txt files are the same!
The local and test files are different. I wonder if that means we can someday get down to one set of requirements files though.

```
43500 37 base.py38.txt
43500 37 base.txt
58672 2 local.py38.txt
28338 2 local.txt
40641 14 test.py38.txt
36194 14 test.txt
```